x-pack/test/ui_capabilities/spaces_only/tests/nav_links.ts

```diff
@@ -45,7 +45,14 @@ export default function navLinksTests({ getService }: FtrProviderContext) {
           case 'nothing_space':
             expect(uiCapabilities.success).to.be(true);
             expect(uiCapabilities.value).to.have.property('navLinks');
-            expect(uiCapabilities.value!.navLinks).to.eql(navLinksBuilder.only('management'));
+            const { kibana, securitySolutionCases, ...navLinksWithoutDeprecated } =
+              uiCapabilities.value!.navLinks;
+            const {
+              kibana: _,
+              securitySolutionCases: __,
```

Comment:
The tests fail otherwise, talking with @azasypkin this is a bug with keeping the old, deprecated permissions around, and will be fixed in an upcoming pr.

Reply:
Thanks, I was not aware!